src/controllers/authController.js

```diff
@@ -0,0 +1,28 @@
+const auth = require('../config/auth');
+
+exports.login = (req, res) => {
+  const token = auth.encodeToken('word');
+  const code = `bearer ${token}`;
+  req.headers.authorization = code;
+  // res.json({ token });
+  res.status(200).send({ code });
+};
+
+exports.logout = (req, res) => {
+  if ((req.headers && req.headers.authorization)) {
+    delete req.headers.authorization;
```

Comment:
why this means he is logged out?

Reply:
I used a bad example for this case. The right way is get the user credential on database and remove his access token. And redirect the login page.